cbmackintosh/overlook
# Feature/booking history

**Clearly and concisely describe the feature.**
- Allows the user to view bookings made for all past dates as well as total bills and expenditures at the hotel
- Adds some other error handling and minor functionality

**Is this a feature or refactor?**
- Feature.

**Problems Encountered & Resulting Solutions**
- None

**Where should the reviewer start?**
scripts.js
- line 32, new function to set the current date upon login. Will be updated before project submission to prevent the user from making a booking on a past date (see line 38)
- line 96, new function to maninpulate the DOM to display past bookings along with bill totals. Requires method added to Guest.js on line 10.

**How should this be tested?**
- Fetch this repo and login to the application using any username/password combination (customer50, overlook2021).
- Click on the 'SHOW BOOKING HISTORY' button and verify that all the displayed information is correct.
- Test out the booking feature to make sure bookings and post requests are still behaving as expected.

**Steps To Take**
- Add functionality to UPCOMING BOOKINGS button
- Add all functionality to manager interface
- Add styling and refactor CSS into Sass
- Write unit tests for class hierarchy
- Add accessibility features and perform accessibility audit
